app/presenters/citable_attributes.rb

```diff
@@ -195,7 +199,7 @@ def csl_type
     end
 
     def abstract
-      work.description.present? ? ActionView::Base.full_sanitizer.sanitize(work.description.join(" ")) : nil
+      work.description.present? ? ActionView::Base.full_sanitizer.sanitize(work.description) : nil
```

Comment:
you changed this from returning a string to returning an array? Comment doc on the method as to what it returns?

Reply:
No, work.description is a string in schihist_digicoll, so the method still returns a string or `nil`. Will add a comment though; this did throw me for a loop on Friday.

Reply:
But yeah, good catch!